src/D2L.Bmx/Authenticate.cs

```diff
@@ -10,7 +11,13 @@ internal class Authenticator {
 		string user,
 		bool nomask,
 		IOktaApi oktaApi ) {
-		// TODO: Add authenticating from cache
+
+		oktaApi.SetOrganization( org );
+
+		var cachedSession = await AuthenticateFromCacheAsync( org, user, oktaApi );
```

Comment:
```suggestion
		OktaAuthenticatedState cachedSession = await AuthenticateFromCacheAsync( org, user, oktaApi );
```

Maybe that or I can rename to `cachedState`

Reply:
for non built in types, especially ones with long names like this, var is better. Its just nice to use type names for primitives like ints/bools to improve readability.